bridges/GithubPackagesBridge.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+class GithubPackagesBridge extends BridgeAbstract
+{
+    const NAME = 'Github Packages Bridge';
+    const MAINTAINER = 'rsd76';
+    const URI = 'https://github.com/';
+    const DESCRIPTION = 'Bridge for getting GitHub packages';
+    const CACHE_TIMEOUT = 15;
+
+    const PARAMETERS = [
+        'Github organization/repository' => [
+            'organization' => [
+                'type' => 'text',
+                'name' => 'Github organization',
+                'exampleValue' => 'RSS-Bridge'
+            ],
+            'repository' => [
+                'type' => 'text',
+                'name' => 'Github repository',
+                'exampleValue' => 'rss-bridge'
+            ],
+            'packagename' => [
+                'type' => 'text',
+                'name' => 'Package name',
+                'exampleValue' => 'rss-bridge'
+            ],
+            'packagetype' => [
+                'type' => 'list',
+                'name' => 'Package Type',
+                'values' => [
+                    'npm' => 'npm',
+                    'maven' => 'maven',
+                    'rubygems' => 'rubegems',
+                    'docker' => 'docker',
```

Comment:
The docker registry doesn't really exist anymore: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-docker-registry
Are all links redirected to the container registry anyway?